crates/test-programs/src/p3/mod.rs

```diff
@@ -0,0 +1,19 @@
+wit_bindgen::generate!({
+    inline: "
+        package wasmtime:test;
+
+        world testp3 {
+            include wasi:cli/imports@0.3.0;
+
+            export wasi:cli/run@0.3.0;
+        }
+    ",
+    path: "../wasi/src/p3/wit",
+    world: "wasmtime:test/testp3",
+    default_bindings_module: "test_programs::p3",
+    pub_export_macro: true,
+    async: [
+        "wasi:cli/run@0.3.0#run",
+    ],
```

Comment:
Could the `wasi:cli/run#run` function be directly marked as `async` to avoid needing to annotate manually here?